README.md

```diff
@@ -264,6 +264,7 @@ Additional lists you might find useful:
 - [Cors plugin](https://github.com/snelg/cakephp-cors) - A lightweight plugin for adding CORS headers to specified endpoints.
 - [FractalTransformerView plugin](https://github.com/andrej-griniuk/cakephp-fractal-transformer-view) - A plugin which allows using [Fractal transformers](http://fractal.thephpleague.com/transformers/) for your API output.
 - [Swagger plugin](https://github.com/alt3/cakephp-swagger) - Swagger plugin for documenting your CakePHP 3.x APIs.
+- [JsonApi plugin](https://github.com/josbeir/cakephp-json-api) - A plugin that implements the [JSON API](http://jsonapi.org/) format.
```

Comment:
You can start with `Implements ...`.
That it is a plugin can be seen from the beginning of that row.
Please also respect the alphabetical order.